packages/schemas/tables/user_logs.sql

```diff
@@ -1,4 +1,4 @@
-create type user_log_type as enum ('SignInUsernameAndPassword', 'SignInEmail', 'ExchangeAccessToken');
+create type user_log_type as enum ('SignInUsernameAndPassword', 'SignInEmail', 'SignInConnector', 'ExchangeAccessToken');
```

Comment:
nit: maybe use `SignInSocial` to keep consistency as we already have `SignInEmail` and `SignInPhone`?

Reply:
updated